public/app/features/dashboard/components/ShareModal/ShareSnapshot.tsx

```diff
@@ -222,25 +220,22 @@ export class ShareSnapshot extends PureComponent<Props, State> {
           </p>
         </div>
         <div className="gf-form-group share-modal-options">
-          <div className="gf-form" ng-if="step === 1">
-            <label className="gf-form-label width-12">Snapshot name</label>
-            <Input width={15} value={snapshotName} onChange={this.onSnapshotNameChange} />
-          </div>
-          <div className="gf-form" ng-if="step === 1">
-            <label className="gf-form-label width-12">Expire</label>
-            <Select width={15} options={expireOptions} value={selectedExpireOption} onChange={this.onExpireChange} />
-          </div>
+          <InlineField labelWidth={24} label="Snapshot name">
```

Comment:
I think it'd be possible to replace `gf-form-group` classes with `InlineFieldRow`. Also would be nice to move the css classes to emotion.